ui/v2.5/src/components/ScenePlayer/vrmode.ts

```diff
@@ -0,0 +1,46 @@
+/* eslint-disable @typescript-eslint/naming-convention */
+import videojs, { VideoJsPlayer } from "video.js";
+import "videojs-vr";
+import * as GQL from "../../core/generated-graphql";
+
+const vrmode = function (this: VideoJsPlayer, scene: GQL.SceneDataFragment) {
+  const player = this;
+
+  switch (scene.projection) {
+    case "SPHERE":
+      switch (scene.stereo_mode) {
+        case "TOP_BOTTOM":
+          player.vr({ projection: "360_TB" });
+          break;
+        case "LEFT_RIGHT":
+          player.vr({ projection: "360_LR" });
+          break;
+        default:
+          player.vr({ projection: "360" });
+      }
+      break;
+    case "DOME":
+    case "MKX200":
+    case "RF52":
+      switch (scene.stereo_mode) {
+        case "LEFT_RIGHT":
+          player.vr({ projection: "180_LR" });
```

Comment:
Have you tested these different projections to see if they work? `videojs-vr` only claims support for 360 projections. From my experience, other projection types just display a black screen?